packages/edit-site/src/components/add-new-pattern/index.js

```diff
@@ -29,6 +33,11 @@ export default function AddNewPattern() {
 		const settings = select( editSiteStore ).getSettings();
 		return !! settings.supportsTemplatePartsMode;
 	}, [] );
+	const { __experimentalCreatePatternFromFile: createPatternFromFile } =
```

Comment:
I think we should probably avoid exporting the `__experimental` prefixed API and instead export it [via private-apis](https://github.com/WordPress/gutenberg/blob/f9e405e0e53d61cd36af4f3b34f2de75874de1e1/docs/contributors/code/coding-guidelines.md#private-selectors-and-actions). What do you think?

Reply:
Oh I wasn't aware of that PR! Just rebased and updated it [5ffa599](https://github.com/WordPress/gutenberg/pull/54337/commits/5ffa599e3e014a882b370a742140dde5ac17aeda)!